nil/contracts/generate.go

```diff
@@ -2,6 +2,8 @@ package contracts
 
 import "embed"
 
+//go:generate python3 genlog.py ./solidity/lib/console.sol ../internal/vm/console/console_generated.go
```

Comment:
It's better to move it directly to Makefile to avoid rebuild on each `make` invocation.